src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java

```diff
@@ -133,6 +132,15 @@
             "com.puppycrawl.tools.checkstyle.checks.coding.VariableDeclarationUsageDistanceCheck",
             "com.puppycrawl.tools.checkstyle.checks.imports.ImportControlCheck"));
 
+    /**
+     *  Inlined configs can not be used in non-java checks, as Inlined config is java style
+     *  multiline comment.
+     *  Such check files needs to be permanently suppressed.
+     */
+    private static final Set<String> PERMANENT_SUPPRESSED_CHECKS = Set.of(
+            // Inlined config is not supported for non java files.
+            "com.puppycrawl.tools.checkstyle.checks.OrderedPropertiesCheck");
```

Comment:
Please do:

```
    private static final Set<String> PERMANENT_SUPPRESSED_CHECKS = Set.of(
            // Inlined config is not supported for non java files.
            "com.puppycrawl.tools.checkstyle.checks.OrderedPropertiesCheck"
    );

```
Since we will be extending this set in the future.

Reply:
done

Reply:
Resolved.